modules/oadp-debugging-oc-cli.adoc

```diff
@@ -34,6 +34,25 @@ $ oc logs pod/<velero>
 [id="oc-velero-debug-logs_{context}"]
 == Velero pod debug logs
 
+You can specify the Velero loglevel in the DataProtectionApplication resource as shown in the example below.
+
+[NOTE]
+====
+If you use an OADP version older than 1.0.3, you have to upgrade it to get access to this option.
+====
+
+[source,terminal]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+spec:
+  configuration:
+    velero:
+      logLevel: warning
+----
+
```

Comment:
```suggestion

Available `logLevel` options are
- trace
- debug
- info
- warning
- error
- fatal
- panic

Use debug for most logging.
```

Reply:
@kaovilai 
Is the original procedure in this section still relevant? ("Use the `oc edit` command to set the `Velero` pod logs to debug level:...") It also sets the log level to `debug`. What is the difference?

Reply:
@kaovilai
Another question: Are there any instructions on upgrading OADP to 1.0.3?

Reply:
The original procedure wouldn't actually work long enough because operator would override the changes.

Until this option was implemented in 1.0.3 you had to have disabled the operator by workaround mentioned in jira. so definitely remove the prior instructions because it doesn't work before without a workaround, and we don't need it now with this change.

Reply:
Upgrade instructions -> "Chapter 4. Administrator tasks OpenShift Container Platform 4.10 | Red Hat Customer Portal" https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/operators/administrator-tasks#olm-upgrading-operators

Reply:
@kaovilai
So this section will only be relevant for OADP 1.0.3, correct?